astropy/cosmology/parameters.py

```diff
@@ -24,10 +24,10 @@
 The list of cosmologies available are given by the tuple
 `available`. Current cosmologies available:
 
-Planck 15 parameters from Planck Collaboration 2015, arXiv: 1502.01589
+Planck 15 parameters from Planck Collaboration 2016, A&A, 594, A13
```

Comment:
they actually tend to get referred to in talks, etc. as Planck 2015, Planck 2013,
unlike, say, WMAP that used WMAP9, etc.  But we can't have a space in the
variable names.

I'll change them to be Planck 2015, Planck 2013